qa/common/gen_qa_models.py

```diff
@@ -2854,6 +2936,112 @@ def create_fixed_models(
             32,
         )
 
+    if FLAGS.tensorrt_bf16:
+        if os.getenv("GPU_MODEL") == "A100":
```

Comment:
Instead of GPU_MODEL, it will be better if we could put a check on compute capability. 
`if CC >= 8.0`

Reply:
Yes, Tanmay. Earlier, didn't follow this approach because we needed to install the library externally. I hope it is fine. I will make the above change. Thank you.